src/views/TabView/TabBarBottom.js

```diff
@@ -51,11 +53,16 @@ type Props = {
   tabStyle?: ViewStyleProp,
   showIcon?: boolean,
   isLandscape: boolean,
+  initialLayout?: Layout,
+  layout: Layout,
+  enableHorizontalTabs: boolean,
 };
 
 const majorVersion = parseInt(Platform.Version, 10);
 const isIos = Platform.OS === 'ios';
-const useHorizontalTabs = majorVersion >= 11 && isIos;
+const canUseHorizontalTabs = majorVersion >= 11 && isIos;
+const isPad =
```

Comment:
Remove usage of `Dimensions` entirely

Reply:
How can I detect device (whether or not is tablet) without use of Dimension?